### PR TITLE
fix(content-server): create storage size limit to uploaded avatar image.

### PR DESCRIPTION
**Pull request by byrw**
Because:

* When the user uploaded a big image and an error message appeared, it was not possible to know the cause of the problem.

This commit:

* Create a limit size to uploaded images.

## Issue that this pull request solves

Closes: #5348, #5123

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/lib/constants.js [hunk 1]
```diff
@@ -86,6 +86,9 @@ module.exports = {
   PROFILE_IMAGE_MIN_WIDTH: 100,
   DEFAULT_PROFILE_IMAGE_MIME_TYPE: 'image/jpeg',
 
+  // Limit to 20 megabytes
+  PROFILE_FILE_IMAGE_MAX_UPLOAD_SIZE: 20 * 1000 * 1000,
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/lib/constants.js:90]: I thought the value for this was going to be [this](https://github.com/mozilla/fxa/blob/main/packages/fxa-profile-server/lib/config.js#L100-L104), but it looks like in production it's actually `1024 * 1024 * 1.35`

Comment (byrw) [reply packages/fxa-content-server/app/scripts/lib/constants.js:90]: I just picked an arbitrary value to be the limit. What number do you think it's better? Should I choose `1.35*1024*1024`?

Comment (jodyheavener) [reply packages/fxa-content-server/app/scripts/lib/constants.js:90]: Yes I believe so, given that’s what we have for the server-side validation.

Comment (byrw) [reply packages/fxa-content-server/app/scripts/lib/constants.js]: If there is a server-side validation, then why it is possible to upload a big image?

Comment (jodyheavener) [reply packages/fxa-content-server/app/scripts/lib/constants.js]: I believe that's where folks are seeing the "System unavailable, try again soon" error message. But it's possible something else is going on that I am unaware of. 

Thank you for updating! I've verified locally that a 3MB file fails, but a 1.2MB file is accepted, as well as some other sizes to be sure.

**Pull request merged**